website/static/css/activity.css

```diff
@@ -1,30 +1,74 @@
 .activity-strip-section >img{
```

Comment:
As we are refactoring the code base, It would be great if you could replace the tabs with spaces and follow the styling rule similar to `website/static/css/main.css`.